00-RELEASENOTES

```diff
@@ -1,8 +1,73 @@
+================================================================================
+Redis 6.2 RC2   Released Mon Jan 11 12:00:00 UTC 2021
+================================================================================
+
+Upgrade urgency LOW: This is the second Release Candidate of Redis 6.2.
+
+Here is a comprehensive list of changes in this release compared to 6.2 RC1,
+each one includes the PR number that added it, so you can get more details
+at https://github.com/redis/redis/pull/<number>
+
+New commands / args:
+* Adds the REV, BYLEX and BYSCORE arguments to ZRANGE, and the ZRANGESTORE command (#7844)
+* Adds the XAUTOCLAIM command (#7973)
+* Adds the MINID trimming strategy and the LIMIT argument to XADD and XTRIM (#8169)
+* Adds the ANY argument to GEOSEARCH and GEORADIUS (#8259)
+* Adds the CH, NX, XX arguments to GEOADD (#8227)
+* Adds the COUNT argument to LPOP and RPOP (#8179)
+* Adds the WRITE argument to CLIENT PAUSE for pausing write commands exclusively (#8170)
+* Changes the proto-ver argument of HELLO to optional (#7377)
+* Adds the CLIENT TRACKINGINFO subcommand (#7309)
+
+Command behavior changes:
+* CLIENT TRACKING errors when given overlapping BCAST prefixes (#8176)
+* SWAPDB invalidates WATCHEed keys (#8239)
+* SORT command behaves differently when used on a writable replica (#8283)
+
+Other behavior changes:
+* Avoids propagating MULTI/EXEC for read-only transactions (#8216)
+* Removes the read-only flag from TIME, ECHO, ROLE, LASTSAVE (#8216)
+* Fixes the command flags of PFDEBUG (#8222)
+* Sentinel: Fixes missing updates to the config file after SENTINEL SET command (#8229)
+
+Bug fixes with compatibility implications (bugs introduced in 6.0):
+* Fixes RDB CRC64 checksum on big-endian systems (#8270)
+* Fixes wrong order of key/value in Lua's map response (#8266)
```

Comment:
@itamarhaber maybe we should be more explicit in these two. please approve.
```suggestion
* Fixes RDB CRC64 checksum on big-endian systems (#8270)
  If you are using bigendian please consider the compatibility implications on RESTORE, replication and
  persistence
* Fixes wrong order of key/value in Lua's map response (#8266)
  If your script uses redis.setresp or returns a map (new features of 6.0), please consider the implications
```

Reply:
Approved and slightly edited (hoping we're < 80 chars)